controllers/clusterversion_controller.go

```diff
@@ -366,6 +383,15 @@ func (c *ClusterVersionReconciler) getOperatorConfig() (*corev1.ConfigMap, error
 	return cm, nil
 }
 
+func (c *ClusterVersionReconciler) getSideCarConfig() (*corev1.ConfigMap, error) {
+	cm := &corev1.ConfigMap{}
+	err := c.Client.Get(c.ctx, types.NamespacedName{Name: csiSidecarConfigMapName, Namespace: c.OperatorNamespace}, cm)
```

Comment:
pls refer the method `get` from other controllers which uses `ctx` and constructs `types.NamespacedName` by default.